src/forms/subscriptions.ts

```diff
@@ -274,20 +277,18 @@ class FormFields extends BaseFormFields {
     // addErrorMessageField adds a text field with a message when a trigger has
     // conditions not supported by the app
     addErrorMessageField(link: string): void {
-        let text = 'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription';
-        text += '\n\n';
-        text += makeBulletedList('Unsupported Fields', this.unsupportedFields);
-        text += '\n\n';
-        text += makeBulletedList('Unsupported Field Operators', this.unsupportedOperators);
-        text += '\n\n' + link;
+        const md = [
+            'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription',
+            makeBulletedList('Unsupported Fields', this.unsupportedFields),
+            makeBulletedList('Unsupported Field Operators', this.unsupportedOperators),
+            '##### Trigger Link',
+            link,
+        ].join('\n');
```

Comment:
You were using before double line breaks, instead of just one. Not sure if this change is intended.

Reply:
The text that was used before was very generic and just thrown into a text box. This new style is fine :)